charts/fleet/values.yaml

```diff
@@ -15,6 +15,28 @@ resources:
     cpu: 0.1 # 100Mhz
     memory: 50Mi
 
+# Node labels for pod assignment
+# ref: https://kubernetes.io/docs/user-guide/node-selection/
+nodeSelector: {}
+
+# Tolerations for pod assignment
+# ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+tolerations: []
+
+# Anti-affinity to disallow deploying client and master nodes on the same worker node
```

Comment:
Do you have a use case for setting the affinity when deploying the chart? As said in @jalseth's previous comment, Fleet does not have a leader-follower model.

If there is a use case for keeping this here, we need to at least update the comment so that it reflects Fleeet's deployment model.

Reply:
@yuriydzobak @jalseth can we come to a conclusion on this?

Reply:
For example, i have a HigLoad POD and i don't want to schedule `fllet` pod with that highload POD. In that case i can configure antiafinity for that. So bassicly, this option not only for scheduling `fleet` if you have more 3 replicas
By default you hardcoed 3 replicas and antiafinity, but somebody want to install all POD on the same worker node

so, if `fleet` doesn't support horizontal autoscale we have to set empty afinity in `values.yaml` and decrease replicas from 3 to 1

Another issue, you hard coded resource in chart, the resource request and limits are empty usually 


from my point of view, the best practicas is always set empty vars in `values.yml` so in this case somebody wants to deploy helm chart in minukube, kined, etc then he faced issues like `resources`, pod scheduling, etc

about ingress, there issue with path `/*` should be `/` becuase i face the issue when ingress can't proxy requests to POD 

Also, there isn't present tls configuration for ingress, and if you are suing cert-manager or self-signer cert, you can't configure it on Ingress side. 

In my opinion, the helm chart should be refactored. Added `_helpers.tpl`, remove hardcoded vars in deployments, etc then add ability to create secrets for mysql and etc, Implement https://github.com/helm/chart-testing
and so on

I've just added a simple fix to start `fleet` in demo mode
When i'll have a time, i'll improve and resolve the critical stuff

Reply:
Okay, I'm good with making `affinity` configurable, please just update the comment to clarify about "client and master nodes" since Fleet doesn't use the model. Happy to review the other concerns when you have a chance. I appreciate both you and @jalseth taking the time to work on Helm as it is new to me. Thanks!

Reply:
I pushed an update to the comment so that we can get this merged.